2_Functions_Part_2.ipynb

```diff
@@ -77,9 +77,20 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        new_list = []
</pre><p>is this line necessary?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RB-Data-Analysis-Python-Course/pfc-week05-sagin132/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='128'/>

Reply:
<p>No? Shouldn't I define a new list first?</p>

Reply:
<p>line 4 is unnecessary, delete it and look change line 5 by a tiny bit so it'll work.</p><p> </p>